### PR TITLE
feat: anonId from app args

### PR DESCRIPTION
**Pull request by NickKhalow**
# Pull Request Description

### Prerequisites
- [ ] List any required setup steps
- [ ] Include environment/configuration requirements

### Test Steps
1. First step
2. Second step
3. Expected result after step 2
4. ...

### Additional Testing Notes
- Note any edge cases to verify
- Mention specific areas that need careful testing
- List known limitations or potential issues

## Quality Checklist
- [ ] Changes have been tested locally
- [ ] Documentation has been updated (if required)
- [ ] Performance impact has been considered
- [ ] For SDK features: Test scene is included

## Code Review Reference
Please review our [Code Review Standards](https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md) before submitting.

### CODE REVIEW

File: Explorer/Assets/DCL/PerformanceAndDiagnostics/Analytics/StaticCommonTraitsPlugin.cs [hunk 1]
```diff
@@ -63,7 +63,20 @@ public override TrackEvent Track(TrackEvent trackEvent)
 
     public struct LauncherTraits
```
Comment (AnsisMalins) [inline Explorer/Assets/DCL/PerformanceAndDiagnostics/Analytics/StaticCommonTraitsPlugin.cs:64]: If all fields are readonly, can make the whole struct readonly.

**Review by AnsisMalins — APPROVED** [review]
